packages/beacon-node/src/api/impl/errors.ts

```diff
@@ -1,6 +1,8 @@
+import {HttpErrorCodes} from "@lodestar/api";
+
 export class ApiError extends Error {
-  statusCode: number;
-  constructor(statusCode: number, message?: string) {
+  statusCode: HttpErrorCodes;
```

Comment:
This change is required due to https://github.com/fastify/fastify/pull/4823, if `number` type is used only a `undefined` payload can be passed to `send()`. Anyhow a good idea to narrow this type here down to only http error codes.